Classes/Domain/Repository/Forum/ForumRepository.php

```diff
@@ -34,13 +34,34 @@
  */
 class ForumRepository extends Repository {
 
+	
 
 	/**
 	 * @var \Mittwald\Typo3Forum\Service\Authentication\AuthenticationServiceInterface
 	 * @inject
 	 */
 	protected $authenticationService = NULL;
 
+	/**
+     * Returns a query for objects of this repository
+     *
+     * @return \TYPO3\CMS\Extbase\Persistence\QueryInterface
+     * @api
+     */
+    public function createQuery() {
+        $query = $this->persistenceManager->createQueryForType($this->objectType);
+        if ($this->defaultOrderings !== array()) {
+            $query->setOrderings($this->defaultOrderings);
+        }
+        if ($this->defaultQuerySettings !== NULL) {
+            $query->setQuerySettings(clone $this->defaultQuerySettings);
+        }
+
+        // don't add sys_language_uid constraint
+        $query->getQuerySettings()->setRespectSysLanguage(FALSE);
```

Comment:
I'm not sure if this is backwards compatible. Maybe there should be some mechanism (TS switch?) to opt-in for this?